emulator/cpu/src/cpu.rs

```diff
@@ -14,21 +14,144 @@ Abstract:
 
 use crate::csr_file::{Csr, CsrFile};
 use crate::instr::Instr;
-use crate::types::{RvInstr, RvMStatus, RvMemAccessType, RvMsecCfg, RvPrivMode};
+use crate::types::{
```

Comment:
I'm a little confused. Can we not just delete this cpu folder now? I think everything we need is in `caliptra-cpu`, so this file should be redundant?